app/containers/ThreadDetails.tsx

```diff
@@ -40,14 +39,29 @@ const styles = StyleSheet.create({
 	}
 });
 
+interface IThreadDetails {
+	item: {
+		tcount: number | string;
+		replies: any;
+		id: string;
+	};
+	user: {
+		id: string
+	};
+	badgeColor: string;
+	toggleFollowThread: Function;
```

Comment:
What's the difference between `Function` and `(): void`?
You used the void option on `TextInput.tsx`, for example.
We should establish a pattern there.

Reply:
I don't know the best way to explain that for you, but basically: 
`(): void` : this is a function with more detail about the return (void in this case), and we can pass the number of the parameters that the function will receive. 
`Function`: this one it's more generic, I use that when we receive a function as a parameter, and I did't find more details.